low-level/framework/rabbitmq/rabbitmq_egress_processor.py

```diff
@@ -255,40 +192,31 @@ def _transmit_msg_on_exchange(self):
                 self._jsonMsg.get("message").get("actuator_response_type").get("thread_controller") is not None):
                 self._add_signature()
-                jsonMsg = json.dumps(self._jsonMsg).encode('utf8')
-                self._ack_connection.publish(exchange=self._exchange_name,
-                                             routing_key=self._ack_routing_key,
-                                             properties=msg_props,
-                                             body=jsonMsg)
+                self._producer.send([json.dumps(self._jsonMsg)])
                 logger.debug("_transmit_msg_on_exchange, Successfully Sent: %s" % jsonMsg)
 
             # Routing requests for IEM msgs sent from the LoggingMsgHandler
             elif self._jsonMsg.get("message").get("IEM_routing") is not None:
                 log_msg = self._jsonMsg.get("message").get("IEM_routing").get("log_msg")
                 if self._iem_route_addr != "":
-                    self._iem_connection.publish(exchange=self._iem_route_exchange_name,
-                                                 routing_key=self._routing_key,
-                                                 properties=msg_props,
-                                                 body=str(log_msg))
+                    self._producer.send([json.dumps(self._jsonMsg)])
                 else:
                     logger.warn("RabbitMQegressProcessor, Attempted to route IEM without a valid 'iem_route_addr' set.")
                 logger.debug("_transmit_msg_on_exchange, Successfully Sent: %s" % log_msg)
             else:
                 self._add_signature()
-                jsonMsg = json.dumps(self._jsonMsg).encode('utf8')
+                jsonMsg = json.dumps(self._jsonMsg)
                 try:
                     if self.store_queue.is_empty():
-                        self._connection.publish(exchange=self._exchange_name,
-                                                routing_key=self._routing_key,
-                                                properties=msg_props,
-                                                body=jsonMsg)
+                        self._producer.send([jsonMsg])                        
                         logger.info(f"Published Alert: {jsonMsg}")
                     else:
-                        self.store_queue.put(jsonMsg)
                         logger.info("'Accumulated msg queue' is not Empty." +\
                                     " Adding the msg to the end of the queue")
-                except connection_exceptions:
-                    logger.error("RabbitMQegressProcessor, _transmit_msg_on_exchange, rabbitmq connectivity lost, adding message to consul %s" % self._jsonMsg)
+                        self.store_queue.put(jsonMsg)
+                except MessageBusError as e:
+                    logger.error(f"RabbitMQegressProcessor, _transmit_msg_on_exchange, error {e} in producing message,\
```

Comment:
reframe these messages as per the renaming

Reply:
That will be changed in remove RabbitMQ references [[EOS-16335](https://jts.seagate.com/browse/EOS-16355)]

Reply:
line too long.